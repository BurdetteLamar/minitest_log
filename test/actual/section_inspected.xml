<log>
  <section_ name='no_inspected'/>
  <section_ name='one_inspected'>
    :one
  </section_>
  <section_ name='more_inspected'>
    :one23.140..2
  </section_>
</log>
