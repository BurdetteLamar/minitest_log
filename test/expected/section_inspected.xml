<log>
  <summary verdicts='0' failures='0' errors='0'/>
  <section name='no_inspected'/>
  <section name='one_inspected'>
    :one
  </section>
  <section name='more_inspected'>
    :one23.140..2
  </section>
</log>
